server/db.js

```diff
@@ -0,0 +1,33 @@
+const mongoose = require('mongoose');
+
+mongoose.connect('mongodb://localhost/profiledb', { useNewUrlParser: true })
+    .then(() => console.log('Connected to MongoDb...'))
+    .catch(err => console.error('Could not connect to MongoDB', err))
+
+const userSchema = new mongoose.Schema({
+    name: String,
+    email: String
+
+})
+
+const Users = mongoose.model('Users', userSchema);
+
```

Comment:
You probably want to keep the logic below in a separate file, e.g inside a 'controllers' directory